sql/sqlbase/structured.go

```diff
@@ -754,6 +754,18 @@ func (desc *TableDescriptor) AddIndex(idx IndexDescriptor, primary bool) error {
 	return nil
 }
 
+// AddColumnToFamily adds the specified column to the specified family.
+// AllocateIDs must be called before the TableDesciptor will be valid.
+func (desc *TableDescriptor) AddColumnToFamily(col string, family string) error {
+	for i := range desc.Families {
+		if desc.Families[i].Name == family {
+			desc.Families[i].ColumnNames = append(desc.Families[i].ColumnNames, col)
+			return nil
+		}
+	}
+	return fmt.Errorf("TODO")
```

Comment:
needs a test with an invalid family name, and should probably have slightly friendlier error message.